src/main/java/seedu/address/logic/commands/CommandResult.java

```diff
@@ -62,6 +66,17 @@ public boolean isExit() {
         return exit;
     }
 
+
+    public boolean isList() {
+        return list;
+    }
+
+    // might need more validation to check if personIndex > -1 before retrieving?
```

Comment:
yes I think it would be better in terms of defensive coding, but it seems like its also handled in ListLogsCommand